src/Symfony/Component/Notifier/Bridge/Expo/ExpoTransportFactory.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Expo;
+
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\AbstractTransportFactory;
+use Symfony\Component\Notifier\Transport\Dsn;
+use Symfony\Component\Notifier\Transport\TransportInterface;
+
+/**
+ * @author Imad ZAIRIG <https://github.com/zairigimad>
+ *
+ * @experimental in 5.4
+ */
+final class ExpoTransportFactory extends AbstractTransportFactory
+{
+    public function create(Dsn $dsn): TransportInterface
+    {
+        $scheme = $dsn->getScheme();
+
+        if ('expo' !== $scheme) {
+            throw new UnsupportedSchemeException($dsn, 'expo', $this->getSupportedSchemes());
+        }
+
+        $token = $dsn->getOption('accessToken');
```

Comment:
what's about using user/password placeholder of the dns to configure this token?

Reply:
I believe Expo recommend to generate accessTokens from the dashboard, like explained here https://docs.expo.dev/accounts/programmatic-access/#personal-account-personal-access-tokens , 

from my experience with Expo Notification, I don't see how to use user/password placeholders for the DNS.

Reply:
have a look to the slack bridge for instance
https://github.com/zairigimad/symfony/blob/fbf4bd2c01807ac5bae4d49bcb58f13ddbc81ceb/src/Symfony/Component/Notifier/Bridge/Slack/SlackTransportFactory.php#L38

Reply:
This is a great example, thank you for point me to this direction, i will try to do it asap and push the fix 👍🏻

Reply:
Done :) thank's for the recommendation